frontend/static/js/pages/profilePage.js

```diff
@@ -187,21 +202,23 @@ export default class extends AView {
 
 		const buttons = this.selectButtons(friendship);
 		const actionBar = buttonBar(buttons);
-		const userPlacement = arrayToElementsList(userData.placements, 'user-placement', smallPlacementCard);
+		const userPlacement = arrayToElementsList(
+			userData.placements,
+			'user-placement',
+			smallPlacementCard
+		);
 		const userDescription = profileDescription(userData.user.description);
 
-		const userStats = friendship === "friend"
-			? scrollContainer(
-				userData.stats,
-				profileStats, "row", "stat-list")
-			: null;
+		const userStats =
+			friendship === 'friend'
```

Comment:
Just a note for myself and us in the future, these equality checks should be done with the friendship enum instead of magic strings!